README.md

```diff
@@ -22,17 +22,17 @@ Found an area of the documentation that needs to be improved? Please raise an [i
 
 Our test-suite performs a validation check on all articles when you open a pull request. You can speed up the review process by running `mvn validate` locally before submitting a pull request.
 
-Should you encounter documentation for features that are deprecated in the minimum eXist-db version mentioned [above](#dependencies), you can simply delete them. If you are unsure about this, please open an [issue](https://github.com/eXist-db/documentation/issues). 
+Should you encounter documentation for features that are deprecated in the minimum eXist-db version mentioned [above](#dependencies), you can simply delete them. If you are unsure about this, please open an [issue](https://github.com/eXist-db/documentation/issues).
 
 ## Building from source
 1.  Clone the repository to your system:
     ```bash
-    $ git clone https://github.com/exist-db/documentation.git exist-documentation
+    $ git clone https://github.com/exist-db/documentation.git documentation
```

Comment:
git clone https://github.com/exist-db/documentation.git
Should be sufficiënt no?

Reply:
good catch

Reply:
I disagree kinda.

Whilst `git clone https://github.com/exist-db/documentation.git documentation` is sufficient. I don't think it is good. If you keep your projects in one directory like many people (me included), when you look through and see `documentation` it is not obvious what it is, which is why I store it locally as `exist-documentation`.

Reply:
Yes but many people will simply clone via the UI menu here on github, and then the folder-name is `documentation`. Folks are free to change it of course on their local system, but for the readme I prefer going with the default behaviour so that the provided command works with both UI or CLI